modules/auxiliary/admin/http/ivanti_vtm_admin.rb

```diff
@@ -39,8 +39,8 @@ def initialize(info = {})
 
     register_options([
       OptString.new('TARGETURI', [true, 'Base path', '/']),
-      OptString.new('NEW_USERNAME', [true, 'Username to be used when creating a new user with admin privileges', Faker::Internet.username]),
-      OptString.new('NEW_PASSWORD', [true, 'Password to be used when creating a new user with admin privileges', Rex::Text.rand_text_alpha(8)]),
+      OptString.new('NEW_USERNAME', [true, 'Username to be used when creating a new user with admin privileges', Faker::Internet.username.gsub('.', '_')]),
```

Comment:
Given your comment earlier I still think you need to sub any characters that don't match the accepted character range with an underscore. So would probably look more like a `[^<accepted characters here>]` type of range.

Reply:
Updated, thank you.